file_formats/ie_formats/spl_v1.htm

```diff
@@ -478,20 +478,23 @@
           <tr>
             <td>0x000d</td>
             <td>1 (char)</td>
-            <td>Resistance<br />
+            <td>Magic Resistance<br />
```

Comment:
same as for itmv1

Reply:
OK, reverted it back to just "Resistance".